### PR TITLE
PR: GitNotes!Mocks

### PR DESCRIPTION
**Pull request by sadainiqrar**

### CODE REVIEW

File: Frontend/Assignments/GitNotes/README.md [hunk 1]
```diff
@@ -12,6 +12,10 @@ Everyone takes notes and sometimes people want to share it with other people, es
   - `Login` menu for login with github
   - `Search` for searching a Note using ID
 
+![Grid View](https://user-images.githubusercontent.com/15946354/62457522-7b1c3600-b794-11e9-9fc1-82e1c5b68828.png)
```
Comment (jabir-hussain) [inline Frontend/Assignments/GitNotes/README.md]: Please at least explain the screen what this screen about and guide the men-tee that what we expect in this screen

File: Frontend/Assignments/GitNotes/README.md [hunk 2]
```diff
@@ -12,6 +12,10 @@ Everyone takes notes and sometimes people want to share it with other people, es
...
+
+![List view](https://user-images.githubusercontent.com/15946354/62457567-9129f680-b794-11e9-9b2e-24d9c6216be6.png)
```
Comment (jabir-hussain) [inline Frontend/Assignments/GitNotes/README.md]: same for each screen

File: Frontend/Assignments/GitNotes/README.md (hunk 1 above)
Comment (sadainiqrar) [reply Frontend/Assignments/GitNotes/README.md]: I've explained them in pages flow & mocks are designed relative to that.

Comment (sadainiqrar) [reply Frontend/Assignments/GitNotes/README.md]: I've updated the content & order of the mocks for better explanation

Comment (jabir-hussain) [reply Frontend/Assignments/GitNotes/README.md]: Great

**Review by jabir-hussain — APPROVED** [review]

**Pull request merged**